lib/powersync/Schema.tsx

```diff
@@ -37,8 +37,25 @@ const eventsV9 = new Table(
   { indexes: {} }
 );
 
+const reschedule_confirmations = new Table(
+  {
+    // id column (text) is automatically included
+    event_id: column.integer,
+    calendar_id: column.integer,
+    original_instance_start_time: column.integer,
+    title: column.text,
+    new_instance_start_time: column.integer,
+    is_in_future: column.integer,
```

Comment:
Why is this an integer and not a boolean? Who at power sink do I have to have a long conversation with lol

Reply:
https://docs.powersync.com/usage/sync-rules/types#postgres-type-mapping
 to quote the powersync docs: 
"There is no dedicated boolean data type. Boolean values are represented as 1 (true) or 0 (false)."